src/NServiceBus.RavenDB.Tests/Persistence/DocumentStoreManagerTests.cs

```diff
@@ -13,12 +13,12 @@ public void Specific_stores_should_mask_default()
         {
             using (var db = new ReusableDB())
             {
-                var settings = new SettingsHolder();
-                settings.Set("Transactions.SuppressDistributedTransactions", true);
-                settings.Set("TypesToScan", new Type[0]);
-                settings.Set("NServiceBus.Routing.EndpointName", "FakeEndpoint");
-                settings.Set("NServiceBus.Transport.TransportInfrastructure", new FakeRavenDBTransportInfrastructure(TransportTransactionMode.None));
-                settings.Set("Endpoint.SendOnly", true);
+                var cfg = new EndpointConfiguration("FakeEndpoint");
+                cfg.UseTransport(new LearningTransport { TransportTransactionMode = TransportTransactionMode.None });
```

Comment:
Is TX mode none needed for the test to pass?

Reply:
Turns out no, was just translating line-by-line, but in the old code those settings had to be faked to work, while with the new transport API it's handled by the constructor. Will remove.